backend/src/models/schemas/anomalies.py

```
@@ -0,0 +1,406 @@
+        valid_metrics = [
+            'runtime_seconds',
+            'credits_consumed',
+            'tokens_used',
+            'executions',
+            'error_rate',
+            'success_rate',
+            'user_activity',
+            'api_latency'
+        ]
+        if v not in valid_metrics:
+            raise ValueError(f"metric_type must be one of {valid_metrics}")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)